src/adminserver/systemcfg/systemcfg.go

```diff
@@ -220,6 +225,11 @@ var (
 		common.ClairURL:                    "CLAIR_URL",
 		common.NotaryURL:                   "NOTARY_URL",
 		common.DatabaseType:                "DATABASE_TYPE",
+		common.ChartRepoURL:                "CHART_SERVICE_URL",
```

Comment:
There are different names for the same thing:
ChartRepo, CHART_SERVICE, ChartMuseum ...

Reply:
Use consistent term chart_repositroy_url